src/helpers/documentHelper.js

```diff
@@ -6,12 +6,15 @@
 /* global Excel, Office, OfficeExtension, Word */
 
 export function writeDataToOfficeDocument(result) {
-  return new OfficeExtension.Promise(function(resolve, reject) {
```

Comment:
Not sure why but was getting browser errors when including OfficeExtension here

Reply:
OfficeExtension.Promise shouldn't be used because webpack is providing a Promise polyfill. Why would you be using when we aren't using it elsewhere? Is this based on older code? This is something that we removed in general a while ago.

Reply:
I can’t recall where this came from honestly.

Reply:
OK, no worries.